source/index.ts

```diff
@@ -150,6 +150,8 @@ is.string = isOfType<string>('string');
 const isNumberType = isOfType<number>('number');
 is.number = (value: unknown): value is number => isNumberType(value) && !is.nan(value);
 
+is.propertyKey = (value: unknown): value is PropertyKey => is.any([is.string, is.number, is.symbol], value);
```

Comment:
Kinda weird place to place it. This should come after all the native types.

Reply:
Would placing it after each `is.plainObject` occurences make you feel more confortable?